editor/import/resource_importer_scene.cpp

```diff
@@ -1161,6 +1162,7 @@ void ResourceImporterScene::get_import_options(List<ImportOption> *r_options, in
 	bool animations_out = p_preset == PRESET_SEPARATE_ANIMATIONS || p_preset == PRESET_SEPARATE_MESHES_AND_ANIMATIONS || p_preset == PRESET_SEPARATE_MATERIALS_AND_ANIMATIONS || p_preset == PRESET_SEPARATE_MESHES_MATERIALS_AND_ANIMATIONS;
 
 	r_options->push_back(ImportOption(PropertyInfo(Variant::REAL, "nodes/root_scale", PROPERTY_HINT_RANGE, "0.001,1000,0.001"), 1.0));
+	r_options->push_back(ImportOption(PropertyInfo(Variant::BOOL, "nodes/meta"), false));
```

Comment:
This added metadata doesn't seem particularly compat breaking, so I guess it could be on by default?

Reply:
I am not familiar with many 3D modeling tools but I can imagine some tools abusing custom properties to store a lot of data (maybe editor plugins?) - so I'm just concerned the user might unknowingly end up with a lot of data in his scene that they never use (and currently not even immediately see as the metadata can be displayed only through a plugin or needs to be inspected by code).

Also if someone already used metadata assuming (performance-wise) there's going to be only their keys/values it might significantly slow down their code if they use some 3D tool that fills the object metadata with a lot of stuff.

But as already stated - I don't really know how things usually work in gamedev/3Dmodeling.